docs/integrations/ledger/ledger_EVM.md

```diff
@@ -0,0 +1,171 @@
+---
+sidebar_position: 1
+title: Ledger EVM
+---
+
+## Using Astar and Shiden EVM ledger apps with MetaMask
+
+## Intro
+
+“Astar EVM” and “Shiden EVM” apps are now available on Ledger hardware wallet devices. This means that MetaMask users can now sign transactions for EVM accounts on those networks using Ledger nano S (plus) or Ledger X devices.
+
+This guide will show you how to set up Astar EVM and Shiden Evm on your Ledger hardware wallet and how to use it in combination with MetaMask.
+
+### Note:
```

Comment:
Since putting hashtags makes a new header entry on the outline, we should either
a) remove this statement since it might be obvious

b) put a > at the beginning to make it a note thingy

eg. 

\> Photos are taken
will look like
> Photos are taken

Reply:
nice one, thanks!